### ISSUE TITLE
Accessibility issue specifically with Internet Explorer 11 and JAWS 

### ISSUE THREAD

**Issue by bwong** [downshift-js/downshift#716]
- `downshift` version: v2.0.0 and up
- `node` version:
- `npm` (or `yarn`) version:

**What you did**:

We were using `downshift v^1` and upgraded to `downshift v^3`.

**What happened**:

When a user using Internet Explorer 11 in conjunction with JAWS (screen reader for PCs) navigates to the input field, they are unable to type anything.  And some how inexplicable any keys the user types initiates shortcuts within the JAWS program.

We first downgraded to `^v2` to see if it was an issue with `v3` but then found the same thing happening on `^v2` so we had to downgraded back to the latest version of `v1`.

**Reproduction repository**:

I'm not sure how to get a repository that reproduces this issue, since I can't even open codesandbox.io on Internet Explorer 11 (a known current issue).

I was able to use the following guide to set up running [JAWS on a Mac](https://joe-watkins.io/accessibility/up-and-running-with-jaws-on-a-mac/).  It's basically running JAWs on a VM on your Mac.

**Problem description**:

A user is unable to type into the input field when using Internet Explorer in conjunction with JAWS.

**Suggested solution**:

Not sure.

**Comment by silviuavram** [comment]
I am not the best with screen readers, no idea if Jaws and IE11 work that well together either. I know there is forms mode in Jaws and scan mode or something like that. Maybe you can try switching between these two and see if you can input stuff.

It may also be the problem of our current HTML structure, which I found not to work on Jaws (but I tested on Chrome): the Jaws focus was behaving very weird.

I am not going to change how Downshift works at the moment in terms of HTML structure. You can make it work like we did in [Stardust Dropdown](https://github.com/stardust-ui/react/tree/master/packages/react/src/components/Dropdown). Here we are also using `getRootProps` in order to match the `exact` HTML structure specified by [ARIA](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html).

**Comment by bwong** [comment]
FYI: I did try it with the forms mode in JAWS and that still could not get input to be entered.  The JAWS focus is behaving very weird.

The newer versions of `downshift` work with other browsers such as Safari and Firefox.  It is just specifically Internet Explorer 11.  Sigh.

I'm not sure what you mean by: 
>You can make it work like we did in Stardust Dropdown. Here we are also using getRootProps in order to match the exact HTML structure specified by ARIA.

Could you elaborate a little more?

**Comment by silviuavram** [comment]
Just make sure when you render your HTML that it matches the ARIA combobox one.
```
<div combobox>
   <input>
</div combobox>
<ul listbox>
```

By default Downshift will have the listbox `ul` inside the combobox `div`. Because of this HTML mismatch the screen reader Jaws may not work :)
